import-service/src/functions/importProducts/handler.ts

```diff
@@ -0,0 +1,40 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { formatJSONResponse } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+import schema from './schema';
+import { PutObjectCommand, S3Client }
+  from "@aws-sdk/client-s3";
+import { getSignedUrl } from '@aws-sdk/s3-request-presigner';
+
+const importProductsFile: ValidatedEventAPIGatewayProxyEvent<typeof schema> = async (event) => {
+  try {
+    const { name } = event.queryStringParameters;
+    const clientParams = {
+      region: 'us-west-2',
+      apiVersion: "2006-03-01",
+      accessKeyId: 'AKIAUJTYHMTUIWAW3FIG',
```

Comment:
You should never put accessKeyId and secretAccessKey into git hub repository!!!
Hide them for example under environment variables

Reply:
I have addressed your comments. Thanks for pointing out keys.